src/css/guide/content.css

```diff
@@ -132,7 +132,7 @@
 	}
 }
 
-@media (min-width: 45rem) {
+@media (min-width: 768px) {
```

Comment:
Well, looking at the ever grewing range of devices I'd not style against arbitrary viewports, but according to content.
Using px unit ignores the font size of the user. Say you defined 24px, because you can't use your glasses right now...